apps/frontend/src/mixins/EvaluationMixin.ts

```diff
@@ -1,14 +1,20 @@
+import {ServerModule} from '@/store/server';
 import {IGroup} from '@heimdall/interfaces';
 import {Component, Vue} from 'vue-property-decorator';
 import {IVuetifyItems} from '../utilities/helper_util';
 
 @Component({})
 export default class EvaluationMixin extends Vue {
   convertGroupsToIVuetifyItems(groups: IGroup[] | undefined): IVuetifyItems[] {
+    const currentUserEmail = ServerModule.userInfo.email;
```

Comment:
is identifying the user via email the correct way to uniquely identify the user?  feels like we should be able to easily just grab their id and use that to filter instead.

Reply:
I 100% agree.  Changed accordingly